docker/run.sh

```diff
@@ -5,6 +5,7 @@
 docker run \
     -ti \
     -p 3000:3000 \
+    -e "SMTP_PASSWORD=smt_password_goes_here" \
```

Comment:
So how do you put the password in here without saving it to the repo?

Reply:
ohh this is just an example to show that env variable can be passed when running built image in demo server